From b9f7e7b9ffb55a4851dfba44734be5cde4791988 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Mon, 15 Mar 2021 17:54:35 +0530
Subject: [PATCH] Fix sepolicy to netd

 From ac6e0b08c91fa7ed2837be1f301b116a1e72192d Mon Sep 17 00:00:00 2001
 From: Steven Fann <steven_fann@realtek.com>
 Date: Tue, 2 Feb 2021 11:22:53 +0530
 Subject: [PATCH] Fix sepolicy to netd.

Allow netd to get adb port from property service.adb.tcp.port

Bug: b/161861298
Test: atest android.net.cts.Ikev2VpnTest#testStartStopVpnProfileV4

Tracked-On: OAM-95330
Signed-off-by: Aiswarya Cyriac <aiswarya.cyriac@intel.com>
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Change-Id: I21588f6e4a7c4587528065c4d3d527ecf44ab993
---
 prebuilts/api/30.0/private/netd.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/prebuilts/api/30.0/private/netd.te b/prebuilts/api/30.0/private/netd.te
index 41473b73d..6e637c14c 100644
--- a/prebuilts/api/30.0/private/netd.te
+++ b/prebuilts/api/30.0/private/netd.te
@@ -17,6 +17,7 @@ allow netd bpfloader:bpf { prog_run map_read map_write };
 # TODO: Remove this permission when 4.9 kernel is deprecated.
 allow netd self:key_socket create;
 
+get_prop(netd, adbd_config_prop)
 get_prop(netd, bpf_progs_loaded_prop)
 
 # Allow netd to write to statsd.
-- 
2.17.1

